### ISSUE TITLE
Can't access gii

### ISSUE THREAD

**Issue by jeremiahtenbrink** [humhub/humhub-modules-devtools#3]
I added the code in the readme to the file /protected/config/web.php and when I try to access gii it tells me  I have taken a wrong turn and that I'm not allowed to access the page. Please help.

**Comment by Felli** [comment]
Read #2 second message. :+1:

**Comment by jeremiahtenbrink** [comment]
Just wanted to say that I ended up getting it working on a development environment at my house using a ubuntu server with lamp installed. I'm not sure if it wasn't working on the web hosting site because I didn't have the correct folder arrangement for yii development or it was because it was on a web server in which I have little control over. I will continue to investigate though.

**Comment by Felli** [comment]
Might be web hosting environment issues because I am having that issue.

**Comment by jeremiahtenbrink** [comment]
After some trial and error. I got gii to work in the web hosting environment by putting in my IPv6 web address into the web.php file. It would not take my IPv4 web address. But I went back to make sure that everything else was the same. So it looks like this issue can be closed for me unless there are others having the same issues. Just thought I would put up my solution so if anyone else is having the same issues they could run into the fix sooner or later. Thanks for all the good work. Love the model generator.  Would it be possible to input a option for the different kinds of navigation links.
